src/ui/src/app/chart-details/chart-details.component.scss

```diff
@@ -3,31 +3,95 @@
 
 .chart-details {
   width: 100%;
-  margin: 3em 0;
-  padding: 0 2em;
   display: flex;
-  flex-direction: column-reverse;
-  flex-wrap: wrap;
+  flex-direction: column;
 
-  @include mappy-bp(medium) {
-    flex-direction: row;
-    flex-wrap: nowrap;
-
-    &__info {
-      padding-left: 1em;
+  &__header {
+    width: 100%;
+    margin-top: -80px;
+    margin-bottom: 40px;
+    &__background {
+      width: 100%;
+      height: 200px;
+      background: md-color($monocular-app-primary);
     }
-
-    &__docs {
-      padding-right: 1em;
+    &__content {
+      padding: 0 1em;
+      max-width: 1000px;
+      margin: auto;
+      margin-top: -75px;
+      display: flex;
     }
-
-    &__info {
-      width: 35%;
+    &__text {
+      flex: 1;
+      h1 {
+        height: 70px;
```

Comment:
Why you force the height of title element? If the user increase the zoom of the page this section won't grow as well